README.md

```diff
@@ -15,3 +15,11 @@ See the [Maven Project Documentation](http://developer.jasig.org/projects/jspc-m
 ## JSP Compilers
 
 The available JSP compilers can be found by browsing [org.jasig.mojo.jspc GroupId](http://search.maven.org/#search%7Cga%7C1%7Cg%3A%22org.jasig.mojo.jspc%22).
+
+### Issues
+
+The plugin is limited to supporting JSP syntax that is supported by the corresponding compiler, which you should make sure also matches the compiler found on your servlet container installation. For any issues refer to the issue tracker for the corresponding compiler.
```

Comment:
>  I felt like this blurb helped clarify that its the compilers themselves that are responsible for JSP syntax issues. For an inexperienced user there still are two problems though. Finding the actual compiler version being used and finding their issue tracker.

Good point.

> I'm not sure how feasible it is to link to issue trackers that might move but maybe it would be useful to explicitly document the compilers versions in the readme? What do you think.

I think that would be feasible.

Maybe taking what you described above and formatting is in markdown could be a good blurb.
Something like:

```markdown
The JSPC plugin is a wrapper around Tomcat's JSP Compiler.
For issues relating to JSP syntax please report the issue to tomcat.
[Tomcat 6 issue tracker](https://bz.apache.org/bugzilla/describecomponents.cgi?product=Tomcat%206) 
[Tomcat 7 issue tracker](https://bz.apache.org/bugzilla/describecomponents.cgi?product=Tomcat%207) 
[Tomcat 8 issue tracker](https://bz.apache.org/bugzilla/describecomponents.cgi?product=Tomcat%208)

Issues relating to maven or configuration options should reported to the [JSPC issue tracker](https://github.com/Jasig/jspc-maven-plugin/issues)
```

Thoughts? :thought_balloon: